### PR TITLE
Fix sizing of text bubbles in message detail view.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/ViewControllers/MessageDetailViewController.swift [hunk 1]
```diff
@@ -343,25 +352,22 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
...
             messageTextProxyView.autoPinHeightToSuperview(withMargin: 10)
             self.messageTextProxyViewHeightConstraint = messageTextProxyView.autoSetDimension(.height, toSize:0)
 
             let row = UIView()
             row.addSubview(bubbleView)
             bubbleView.autoPinHeightToSuperview()
-            bubbleView.autoPinLeadingToSuperview(withMargin: 10)
-            bubbleView.autoPinTrailingToSuperview(withMargin: 10)
+            bubbleView.autoPinEdge(toSuperviewEdge: isIncoming ? .leading : .trailing, withInset: bubbleViewHMargin)
+            self.bubbleViewWidthConstraint = bubbleView.autoSetDimension(.width, toSize:0)
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/MessageDetailViewController.swift]: Why are we setting the width to 0?

File: Signal/src/ViewControllers/MessageDetailViewController.swift [hunk 2]
```diff
@@ -617,52 +623,72 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
...
+        guard let bubbleViewWidthConstraint = bubbleViewWidthConstraint else {
+            owsFail("\(TAG) Missing bubbleViewWidthConstraint")
             return
         }
 
         if messageTextView.width() != messageTextProxyView.width() {
             owsFail("\(TAG) messageTextView.width \(messageTextView.width) != messageTextProxyView.width \(messageTextProxyView.width)")
         }
 
+        let bubbleViewHMargin: CGFloat = 10
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/MessageDetailViewController.swift]: Can we dedupe these to use the equivalent constants extracted around line 310

Comment (charlesmchen) [reply Signal/src/ViewControllers/MessageDetailViewController.swift]: Removed!

File: Signal/src/ViewControllers/MessageDetailViewController.swift (hunk 1 above)
Comment (charlesmchen) [reply Signal/src/ViewControllers/MessageDetailViewController.swift]: This is a placeholder value. We set the actual value in `updateTextLayout()` which does measurement, etc..

**Pull request merged**